.travis.yml

```diff
@@ -27,6 +28,10 @@ before_install:
   - gem update --system
   - gem update bundler
 
+before_script:
+  # Patch for Ruby 2.4.0 compatibility.
+  - ./scripts/patch_simplecov_0_12_0
+
```

Comment:
As a reference, running `./scripts/patch_simplecov_0_12_0` is same with patching https://github.com/colszowka/simplecov/commit/050eb68